.github/workflows/README.md

```diff
@@ -0,0 +1,57 @@
+# GitHub Workflows
+
+This directory contains the GitHub Actions workflows for the kgateway.dev website.
+
+## update-api-docs.yml
+
+This workflow is used to update the API and Helm reference docs for the kgateway.dev website, based on code in the kgateway repo.
+
+### API reference
+
+[Published page](https://kgateway.dev/docs/reference/api/)
+
+Source files: 
+* [Go code in the API directory](https://github.com/kgateway-dev/kgateway/tree/main/api/v1alpha1)
+
+The workflow uses the [crd-ref-docs](https://github.com/elastic/crd-ref-docs) tool to generate Markdown documentation from the API types.
+
+Process to generate markdown content from source files:
+
+1. Update the comments in the .go files in the kgateway/api directory
+2. Run `make generated-code`
+3. Open a PR and merge your changes into the code repo
+4. In the kgateway.dev docs repo, manually trigger the Update API Documentation GitHub Action
+5. The Action updates the [markdown source file](https://github.com/kgateway-dev/kgateway.dev/blob/main/content/docs/reference/api.md)
+6. The Action opens a PR with the updated content
+7. Review and merge the PR
+8. Upon merge, the docs should be automatically built and published
+
+### Helm reference
+
+[Published page](https://kgateway.dev/docs/reference/helm/)
+
+Source files:
+* [Helm files in the code repo](https://github.com/kgateway-dev/kgateway/tree/main/install/helm)
+
+The workflow uses the [helm docs](https://github.com/norwoodj/helm-docs/) tool to generate Markdown documentation from the API types.
+
+Process to generate markdown content from source files: (same as the API docs process)
```

Comment:
do we need `(same as the API docs process)`?

Reply:
```suggestion
Process to generate markdown content from source files:
```